service/service.go

```diff
@@ -83,7 +83,8 @@ func Run(ctx context.Context, cfg *config.Config, serviceList *ExternalServiceLi
 	deprecatedESClient := elasticsearch.New(cfg.ElasticSearchAPIURL, elasticHTTPClient, cfg.SignElasticsearchRequests, esSigner, cfg.AwsRegion, cfg.AwsService)
 
 	// Initialise query builder
-	queryBuilder, err := query.NewQueryBuilder()
+	pathToTemplates := ""
```

Comment:
Probably worth making this a global constant: `const pathToTemplates = ""` below imports